modules/nf-core/drep/compare/main.nf

```diff
@@ -0,0 +1,49 @@
+process DREP_COMPARE {
+    tag "${meta.id}"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
+        ? 'https://depot.galaxyproject.org/singularity/drep:3.5.0--pyhdfd78af_0'
+        : 'biocontainers/drep:3.5.0--pyhdfd78af_0'}"
+
+    input:
+    tuple val(meta), path("fastas/*")
+
+    output:
+    tuple val(meta), path("${prefix}"), emit: directory
```

Comment:
I understand that getting the complete directory is convenient and I am not aware of current standards. However, having output files in individual channels seems to me the more reliable method, so I would recommend that.

Reply:
According to the documentation, you pass the entire directory to `drep dereplciate`, so given this is only the firs step in that process it makes sense to export that - the vast majority of the files in the subdirectories appear to be intermediate files (with nasty inconsistent names).

But  I can update to export the image fdiles and logs separately too

Reply:
Ah I see, just the first step. Sure that an excellent reason to capture the whole directory!

Reply:
I also realised it doesn't make sense to make output channels for PDFs or the log anyway as you wouldn't use it in any downstream processes